core/tests/protractor/stateEditor.js

```diff
@@ -134,6 +134,30 @@ describe('State editor', function() {
     users.logout();
   });
 
+  it('should skip the customization modal for interactions having no ' +
```

Comment:
Isn't the process of changing the interactions same for both type of interactions? After I've set an interaction, the process of changing it is to first delete the interaction, then select a new interaction by clicking add interaction.